lib/model/announcement/announcement.dart

```diff
@@ -7,22 +7,35 @@ class Announcement extends ChatThread {
   final Map<String, dynamic> additionalData;
   final DateTime startedAt;
   final DateTime endedAt;
+  final String? imageURL;
+  final String? notificationType;
+  final String? deliveryTimeOfDay;
+  final bool inAppEnabled;
 
   Announcement({
     required this.announcementContentId,
     required this.content,
     required this.additionalData,
     required this.startedAt,
     required this.endedAt,
+    required this.inAppEnabled,
+    required this.notificationType,
```

Comment:
could you please make it as enum

Reply:
that isn't use in this card, but will have an enum in next card